install/mysql/glpi-empty.sql

```diff
@@ -2956,14 +2956,15 @@ CREATE TABLE `glpi_entities` (
   `date_mod` timestamp NULL DEFAULT NULL,
   `date_creation` timestamp NULL DEFAULT NULL,
   `autofill_decommission_date` varchar(255) COLLATE utf8_unicode_ci NOT NULL DEFAULT '-2',
+  `suppliers_as_private` int(11) NOT NULL DEFAULT '-2',
```

Comment:
Note to myself: must be reported on https://github.com/glpi-project/glpi/pull/5947